src/login/styles/_inputs.scss

```diff
@@ -122,6 +124,33 @@ input:focus {
   letter-spacing: 0.025rem;
 }
 
+.label-helpblock-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+
+  label{
+    font-size: 1rem;
+    margin-left: -2px;
+  }
+}
+
+.help-block {
+  font-size: 1rem;
+  text-transform: lowercase;
+  padding-bottom: 3px;
+}
+
+.eduid-field-error {
```

Comment:
I know most of this naming is legacy code, but as we are updating it could we rename these to be more descriptive? (i.e. can we find something more descriptive as a class for the helpBlock?)